src/modules/KYB/containers/PendingModal/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
`PenfingModal`, `RejectModal`, and `SuccessModal` is the same with different `icon` and `title`, and `subtitle`. Can you merge it into one container to accept different props?

Reply:
I intentionally separated them. I think we are going to need more functionality on each of them